src/Player.cpp

```diff
@@ -0,0 +1,26 @@
+#include "Player.hpp"
+
+Player::Player(const std::string &color)
+        : color_(color),
+        money_(3000),
+        location_(0)
+{}
+
+std::string Player:: getColor() const {return color_;}
+int Player:: amountOfMoney() const {return money_;}
+int Player :: getLocation() const { return location_;}
+bool Player :: ownsProperty() const
+{
+    if(properties_.empty()) return false;
```

Comment:
This could be written in a one line:
`return not properties_.empty()`
BTW Name for this function is rather: hasAnyProperty. For ownsProperty, I would expect to get one argument and check if the player owns a given property.